src/nativeMain/kotlin/api/EclairClient.kt

```diff
@@ -39,9 +43,19 @@ class EclairClient(private val apiHost: String, private val apiPassword: String)
         }
     }
 
+    private val multiPartHttpClient = HttpClient(CIO) {
+        install(Auth) {
+            basic {
+                credentials {
+                    BasicAuthCredentials(username = "", password = apiPassword)
+                }
+            }
+        }
+    }
```

Comment:
This isn't necessary, I removed it and just used the `httpClient` above, and I'm able to successfully use the `connect` command. I don't understand why you're saying it doesn't work for you?

Reply:
Probably it was due to the other reasons(which you stated above). I just tried to do it this way, in case if it works.